gradlew

```diff
@@ -33,11 +33,11 @@ DEFAULT_JVM_OPTS=""
 # Use the maximum available, or set MAX_FD != -1 to use that value.
 MAX_FD="maximum"
 
-warn () {
+warn ( ) {
```

Comment:
How come this change? Looks spurious to me.

Reply:
This is from running the command `$ gradle wrappers`. May be due to different style configs between Gradle versions.